src/resources/views/layout/layout.blade.php

```diff
@@ -0,0 +1,8 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.min.js"></script>
+
+</head>
```

Comment:
Already have a layout folder with a layout file again? Is this still in WIP with Karl?

Reply:
Yes po sir, sa end po ni Karl maayos na po yung files under layout folder. Will fix po after merging baka po kasi magka conflict. Thank you po.